pkg/gce-pd-csi-driver/node_test.go

```diff
@@ -350,6 +350,71 @@ func TestNodeGetVolumeLimits(t *testing.T) {
 			machineType:    "c4d-standard-32",
 			expVolumeLimit: 31,
 		},
+		{
+			name:           "c3-highcpu-192-metal",
+			machineType:    "c3-highcpu-192-metal",
+			expVolumeLimit: c3MetalHyperdiskLimit,
+		},
+		{
+			name:           "c3-standard-192-metal",
+			machineType:    "c3-standard-192-metal",
+			expVolumeLimit: c3MetalHyperdiskLimit,
+		},
+		{
+			name:           "c3-highmem-192-metal",
```

Comment:
Are we planning to backport this? Since this is way lower than the 128(for both PD and Hyperdisk), backport request will be rejected.

Reply:
But metal is new, so this doesn't affect existing workloads?

Reply:
NVM it does not support PD at all.